File: fix_remaining_missing_imports.js
const fs = require('fs');

const filesToFix = [
  'tests/plot-outcomes.test.ts',
  'tests/quest-kill-pressure.test.ts'
];

for (const file of filesToFix) {
  let content = fs.readFileSync(file, 'utf8');
  if (!content.includes('import { getPlotNpcCount }')) {
    content = "import { getPlotNpcCount } from '../src/data/npc_packages';\n" + content;
    fs.writeFileSync(file, content);
    console.log('Added import to', file);
  }
}

File: fix_remaining_missing_imports.py
import sys
files = [
  'tests/plot-outcomes.test.ts',
  'tests/quest-kill-pressure.test.ts'
]

for file in files:
    with open(file, 'r', encoding='utf-8') as f:
        content = f.read()
    if "import { getPlotNpcCount }" not in content and "getPlotNpcCount" in content:
        content = "import { getPlotNpcCount } from '../src/data/npc_packages';\n" + content
        with open(file, 'w', encoding='utf-8') as f:
            f.write(content)
        print("Fixed", file)

